.github/workflows/ci.yml

```diff
@@ -10,57 +10,138 @@ on:
   pull_request:
   workflow_dispatch:
 
+env:
+  IMAGE_NAME: openformulieren/open-forms-ext-stuf-zds-payments
+
 jobs:
   tests:
+    name: Run the Django test suite
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python: ['3.10', '3.11', '3.12']
-        django: ['4.2']
-
-    name: Run the test suite (Python ${{ matrix.python }}, Django ${{ matrix.django }})
+    services:
+      postgres:
+        image: postgres:14
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        # Needed because the postgres container does not provide a healthcheck
+        options:
+          --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+          --name postgres
+      redis:
+        image: redis:6
+        ports:
+          - 6379:6379
 
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
+      - name: Checkout Open Forms
+        uses: actions/checkout@v3
+        with:
+          repository: open-formulieren/open-forms
+          path: open-forms
+
+      - name: Checkout StUF-ZDS payments extension
+        uses: actions/checkout@v3
+        with:
+          path: extension
+
+      - name: Set up backend environment
+        uses: maykinmedia/setup-django-backend@v1.1
         with:
-          python-version: ${{ matrix.python }}
+          apt-packages: 'libxml2-dev libxmlsec1-dev libxmlsec1-openssl gettext postgresql-client gdal-bin'
```

Comment:
Having `libxml2-dev` and `libxmlsec1-dev` present has a nasty side-effect that it breaks the uwsgi binary. They're not needed anymore in latest versions of Open Forms, since we can install the binary wheels these days of `lxml` and `xmlsec`.

See https://github.com/open-formulieren/open-forms/commit/6882bd292b531db8872da103a131dcd2e67e5a6e for the patch in OF itself.